packages/asc-ui/src/components/Link/LinkStyle.ts

```diff
@@ -1,18 +1,66 @@
-import styled, { css } from '@datapunt/asc-core'
+import styled, { css, Theme } from '@datapunt/asc-core'
 import { color } from '../../utils'
-import { focusStyleText } from '../../utils/themeUtils'
-import Typography from '../Typography'
+import { focusStyleText, svgFill } from '../../utils/themeUtils'
+import Typography, { TypographyProps } from '../Typography'
+import IconStyle from '../Icon/IconStyle'
 
-export const LinkStyleCSS = css`
-  color: ${color('primary')};
+export const NoStyleCSS = css`
   display: inline-block;
-  ${focusStyleText()}
+  text-decoration: none;
+  color: inherit;
+`
+export const InlineStyleCSS = ({
+  theme,
+}: {
+  theme: Theme.ThemeInterface
+}) => css`
+  display: inline-block;
+  color: ${color('primary')({ theme })};
 
   &:hover {
     color: ${color('secondary')};
   }
 `
+export const LinkStyleCSS = ({ theme }: { theme: Theme.ThemeInterface }) => css`
+  display: block;
+  padding-left: 15px;
+  color: ${color('tint', 'level6')({ theme })};
+  text-decoration: none;
+  font-weight: 700;
+
+  &:hover {
+    color: ${color('secondary')({ theme })};
+    text-decoration: underline;
+    ${IconStyle} {
+      ${svgFill('secondary')({ theme })};
+    }
+  }
+`
+
+type TypeLink = 'list' | 'inline' | 'nostyle'
+
+export const getLinkType = (linkType: TypeLink = 'inline') => ({
+  theme,
+}: {
+  theme: Theme.ThemeInterface
+}) => {
+  switch (linkType) {
+    case 'nostyle':
+      return NoStyleCSS
+    case 'inline':
+      return InlineStyleCSS({ theme })
+    case 'list':
```

Comment:
and rename this to default? as this is used almost everywhere...

Reply:
ok